Hunting Queries/DnsEvents/DNS_Tunneling.txt

```diff
@@ -0,0 +1,50 @@
+// Name: DNS Tunneling
+// Author: Vani Asawa
+//
+// Description: 
+// Searches for second levels domains (slds) within an IP that might be impacted by tunneling, based on mathematically computed subdomain thresholds
```

Comment:
Let's try to include more details about what a hit means and how to investigate it further.  The result set can include many domains, as in my testing and many customers may not understand how to investigate each hit.  I also saw hits for wpad.domain.com, I do not think we want to include those.